### PR TITLE
Azure DSVM Instructions

### PR DESCRIPTION
**Pull request by gopitk**
@hamelsmu - Can you take a look and merge if this looks good?

### CODE REVIEW

**Comment by hamelsmu** [comment]
Thanks won’t have time to look at this for a couple of weeks ~ mid October


If someone else has the time to review this great the otherwise I’ll return
to this around that time

**Comment by gopitk** [comment]
@jph00  - Can you please take a look? I have used this instructions earlier and have verified the steps today as I was writing the instructions.

File: docs/start_azure_dsvm.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# Azure Data Science Virtual Machine
+
+This is a quick guide to getting started with fast.ai Deep Learning for Coders course on [Microsoft Azure](https://azure.microsoft.com/) cloud. We will use the Azure [Data Science Virtual Machine (DSVM)](http://aka.ms/dsvm) which is a family of Azure Virtual Machine images, pre-configured with several popular tools that are commonly used for data analytics, machine learning and AI development. 
+
+## Pricing
+The cheapest GPU configuration is a Standard_NC6 instance in Azure which has one NVidia Tesla K80 GPU and six CPU cores. This instance will incur about $0.40 per hour of [compute charges](https://azure.microsoft.com/en-us/pricing/details/virtual-machines/linux/) for a dedicated Standard_NC6_Promo instance, or **$0.13** per hour if you use [Spot (pre-emptable) Standard_NC6 instances](https://azure.microsoft.com/en-us/pricing/details/virtual-machine-scale-sets/linux/).
```
Comment (jph00) [inline docs/start_azure_dsvm.md:6]: NC6 is $0.90, not $0.40. But I don't think it's a great choice anyway. I recommend `NC4as T4 v3`, which is $0.52 PAYG or $0.21 spot.

**Review by jph00 — CHANGES_REQUESTED** [review]
Great! Just one suggested change

File: docs/start_azure_dsvm.md (hunk 1 above)
Comment (gopitk) [reply docs/start_azure_dsvm.md:6]: Thanks Jeremy. Standard_NC6_Promo (which is the same as Standard_NC6 with K80 except it is only available as dedicated i.e no Spot pricing and no 1 year reserved price) is $0.40 per pricing page. Yes. T4 instances would be good. But it is only available on limited preview  and not widely available yet for Azure customers yet and has extra friction to fill out a form and likely not get approved initially as it is in only one datacenter now. When it is more generally available we can switch to recommend that option. Hope that works.

Comment (jph00) [reply docs/start_azure_dsvm.md:6]: Got it - thanks

**Pull request merged**